.github/workflows/docker.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
           - name: Reuse ccache directory
             uses: actions/cache@v4
             with:
-                path: .ccache
+                path: ~/.cache/ccache
```

Comment:
Are you sure that it is correct directory for docker? Docker builds started takes more time than in develop branch
docker-compose expected ccache in: https://github.com/userver-framework/userver/blob/develop/.github/docker-compose.yml#L10

Reply:
yep. you're totally right. docker build worked correctly. i reverted them. but i dont have any idea for conan. it's start always without cache.

Reply:
We'll try to cope with conan ourselves. Probably, it does not need the ccache at all
I'll merge your PR tomorrow, it is really speed ups the build, thanks!